### ISSUE TITLE
SelectControl misses a `selected` attribute for the selected option

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#16836]
Splitting this out from https://github.com/WordPress/gutenberg/pull/15976#issuecomment-516790939

#15976 introduced the ability to set a `disabled` attribute on a select option. So far so good.

A consideration was made on the specific usage of a disabled option in the Tag Cloud taxonomy select:

<img width="314" alt="Screenshot 2019-07-31 at 15 04 56" src="https://user-images.githubusercontent.com/1682452/62214120-aeca1b00-b3a4-11e9-8654-f8cc08c5e070.png">

As the disabled `- Select - ` option is only visible when opening the select dropdown, I'm not sure I understand its actual purpose from a UI perspective.

An observation was made that without disabling the first option, browsers select the first option on page load. However, that's because the actually selected option misses `selected` attribute. As far as I can tell, there's nothing in the code to support the `selected` attribute.

To recap:
- the selected option misses a `selected` attribute
- according to the HTML 5.2 spec, the [selected attribute](https://www.w3.org/TR/html52/sec-forms.html#the-option-element) is specifically meant to indicate whether the option is selected by default
- on page load, browsers will understand what the initially selected option is thanks to the `selected` attribute
- the code should take into account the `selected` attribute and update it as necessary

Once the `selected` attribute will work properly, I'd like to propose to remove the disabled `- Select -` first option from the Tag Cloud dropdown, as it serves no purpose.

**Comment by scarabcoder** [comment]
It's not currently documented, but you can use the `value` attribute to set the default selected attribute. It's in the component readme but it's not in the documentation for some reason.

**Comment by Baelx** [comment]
Any progress on either adding a "selected" declaration or updating the docs? I'm not the original reporter but I've looked through the README.md for the workaround suggested by @scarabcoder and it's not really clear how it's achieved with "value."

**Comment by Mamaduka** [comment]
Closing the issue. The `value` looks to be correctly documented now - https://developer.wordpress.org/block-editor/reference-guides/components/select-control/#props.

The Storybook also makes it easier to experiment with components - https://wordpress.github.io/gutenberg/?path=/docs/components-selectcontrol--docs.
